js/index.js

```diff
@@ -0,0 +1,32 @@
+const today = new Date()
+const thisYear = today.getFullYear()
+console.log(today);
+console.log(thisYear);
+const footer = document.querySelector('footer');
+console.log(footer);
+const copyright = document.createElement('p');
+console.log(copyright)
+copyright.innerHTML = 'Jessica Baker 2022'
+document.body.appendChild(copyright)
```

Comment:
This works fine, since you're adding the copyright `p` tag to the end of the body; but to stick with what the instructions asked for: 
```Using "DOM Manipulation", append the `copyright` element to the footer```, 
we should be appending this not to `document.body` but to the `footer` that you saved into a variable on line 5.  _Then_ you can append the whole `footer` to the body.